### ISSUE TITLE
Not able to start application

### ISSUE THREAD

**Issue by sunzoje** [profclems/glab#232]
**Description**
With the commit sha dec7d94f48ed9cef2d59b1c4499dcd1d7853bb21 I'm getting
```
2020/09/29 13:50:01 error loading config: "expected a top level map"
```
when trying to start the application. Every command with flags displays the aforementioned message.

**Expected Behavior**
Application starts without aforementioned error message.

**Actual Behavior**
Aforementioned error message is displayed.

**Steps to Reproduce**
```
git clone https://github.com/profclems/glab
cd glab
git checkout dec7d94
make build
./bin/glab --version
```

**Your Environment**
* Version used: sha dec7d94f48ed9cef2d59b1c4499dcd1d7853bb21
* Operating System and version: macOS Catalina 10.15.7

**Comment by profclems** [comment]
Sorry you've had this issue... Were you having an older version installed initially?

**Comment by sunzoje** [comment]
No. No issue at all.

**Comment by sunzoje** [comment]
Closing. Problem was due to old global config structure. Fixed as of sha 1b77f9e259a6337d616e8dfc2eec1dae6d3dddd5
